src/argilla/client/feedback/schemas.py

```diff
@@ -452,3 +475,253 @@ class FeedbackDatasetConfig(BaseModel):
+                majority_value = counter.most_common(1)[0][0]
+
+            rec.unified_responses = {field.name: [UnificationValueSchema(value=majority_value, strategy=self.value)]}
+        return rec
+
+    def _majority_weighted(self, records: List[FeedbackRecord], field: LabelQuestion):
+        raise NotImplementedError("Not implemented yet")
+
+
+class MultiLabelQuestionStrategy(LabelQuestionStrategyMixin, Enum):
+    """
+    Options:
+        - "majority": the majority value of the labels
+        - "majority_weighted": the majority value of the labels, weighted by annotator's confidence
+        - "disagreement": preserve the natural disagreement between annotators
+    """
+
+    MAJORITY: str = "majority"
+    MAJORITY_WEIGHTED: str = "majority_weighted"
+    DISAGREEMENT: str = "disagreement"
+
+    def _majority(self, records: List[FeedbackRecord], field: MultiLabelQuestion):
+        for rec in records:
+            counter = Counter()
+            # only allow for submitted responses
+            responses = [resp for resp in rec.responses if resp.status == "submitted"]
+            # get responses with a value that is most frequent
+            for resp in responses:
+                if isinstance(resp.values[field.name].value, list):
+                    for value in resp.values[field.name].value:
+                        counter.update([value])
+                else:
+                    counter.update([resp.values[field.name].value])
+            # check if there is a majority based on the number of responses
+            majority = int(len(responses) // 2) + 1
+            majority_value = []
+            for value, count in counter.items():
+                if count >= majority:
+                    majority_value.append(value)
+            rec.unified_responses = {field.name: UnificationValueSchema(value=majority_value, strategy=self.value)}
+        return records
+
+    def _majority_weighted(self, records: List[FeedbackRecord], field: MultiLabelQuestion):
+        raise NotImplementedError("Not implemented yet")
+
+
+class TrainingDataForTextClassification(BaseModel):
+    """Training data for text classification
+
+    Args:
+        text: TextField
+        label: Union[RatingQuestion, LabelQuestion, MultiLabelQuestion]
+        label_strategy: Union[RatingQuestionStrategy, LabelQuestionStrategy] = None
+
+    Examples:
+        >>> from argilla import TextField, LabelQuestion, LabelQuestionStrategy, TrainingDataForTextClassification
+        >>> dataset = rg.FeedbackDataset.from_argilla(argilla_id="...")
+        >>> training_data = TrainingDataForTextClassification(
+        ...     text=dataset.fields[0],
+        ...     label=dataset.questions[0],
+        ...     label_strategy=LabelQuestionStrategy.MAJORITY
+        ... )
+
+    """
+
+    text: TextField
+    label: Union[str, RatingQuestion, LabelQuestion, MultiLabelQuestion]
+    label_strategy: Union[RatingQuestionStrategy, LabelQuestionStrategy] = None
```

Comment:
We can group label and label strategy in a BaseModel as :
```python
    class Rating(BaseModel):
        question: RatingQuestion
        strategy: RatingQuestionStrategy = RatingQuestionStrategy.MEAN

    class Label(BaseModel):
        question: Union[LabelQuestion, MultiLabelQuestion]
        strategy: LabelQuestionStrategy = LabelQuestionStrategy.MAJORITY

The, we can use them in the training model definition and reduce the root validator logic:
```suggestion
    text: TextField
    label: Union[Rating, Label]
```